docs/api/IoT.js-API-DNS.md

```diff
@@ -6,61 +6,78 @@ The following shows dns module APIs available for each platform.
 | :---: | :---: | :---: | :---: |
 | dns.lookup | O | O | X |
 
-※ dns.lookup on `nuttx` works but due to lack of `nuttx` implementation, internally it only allows valid `ip4` address,
-but we are going to support `ip6` address, soon.
-
-
-### Contents
-
-- [DNS](#dns)
-    - [Properties](#properties)
-        - [`dns.ADDRCONFIG`](#dnsaddrconfig)
-        - [`dns.V4MAPPED`](#dnsv4mapped)
-    - [Module Functions](#module-functions)
-        - [dns.lookup(hostname[, options], callback)](#dnslookuphostname-options-callback)
+※ dns.lookup currently only returns IPv4 addresses. Support for IPv6 addresses are on the roadmap.
 
 
 # DNS
 
+The `dns` module provides a method to perform host name resolution. The functionallity is
+implemented on top of the underlying operation system facilities. Please see
+[Implementation considerations section](#implementation-considerations).
 
 ## Properties
 
+The following properties are flags which can be passed as hints for the
+[`dns.lookup()`](#dnslookuphostname-options-callback) method.
 
 ### `dns.ADDRCONFIG`
-* `<Number>`
+* `{Number}`
 
 Returned address types are determined by the types of addresses supported by the current system.
 
 
 ### `dns.V4MAPPED`
-* `<Number>`
+* `{Number}`
 
 If the IPv6 family was specified, but no IPv6 addresses were found, then return IPv4 mapped IPv6 addresses. 
 
 
-## Module Functions
+## Module Methods
 
 
 ### `dns.lookup(hostname[, options], callback)`
-* `hostname <String>`
-* `options <Object> | <Number>`
-  * `family <Number>` - The record family. Must be 4 or 6. IPv4 and IPv6 addresses are both returned by default.
-  * `hints <Number>` - One or more supported getaddrinfo flags. Multiple flags may be passed by bitwise ORing their values.
-* `callback <Function(err: Error, address: String, family: Number)>`
-  * `err <Error>`
-  * `address <String>` - A string representation of an IPv4 or IPv6 address.
-  * `family <Number>` - 4 or 6, denoting the family of address.
+* `hostname {String}` Hostname to be resolved.
+* `options {Object> | <Number>}`
```

Comment:
I understand that it's hard to find bracket missing mistakes like this. I have no idea to validate this. Maybe we could use `https://github.com/nodejs/doc-tool` and check the result.